lerna.json

```diff
@@ -1,16 +1,14 @@
 {
-  "packages": [
-    "packages/*"
-  ],
+  "packages": ["packages/*"],
   "version": "independent",
   "npmClient": "yarn",
-  "useWorkspaces": true,
```

Comment:
Just curious why remove this? Does "packages" override it?

Reply:
`useWrokspaces` is true by default now.
<img width="1147" alt="image" src="https://github.com/amplitude/Amplitude-TypeScript/assets/31029607/5c055f8f-b063-4878-8f8c-567eb35ccc16">